diff --git a/Makefile b/Makefile
index ceb9d77..aa88b5b 100644
--- a/Makefile
+++ b/Makefile
@@ -59,7 +59,7 @@ SOMAJOR=2
 SOMINOR=1
 SORELEASE?=.0#   Declare empty to leave out from library file name.
 
-MINISAT_CXXFLAGS = -I. -D __STDC_LIMIT_MACROS -D __STDC_FORMAT_MACROS -Wall -Wno-parentheses -Wextra
+MINISAT_CXXFLAGS = -I. -D __STDC_LIMIT_MACROS -D __STDC_FORMAT_MACROS -Wall -Wno-parentheses -Wextra -fpermissive
 MINISAT_LDFLAGS  = -Wall -lz
 
 ECHO=@echo
